optimizely/event/processor_test.go

```diff
@@ -16,7 +16,7 @@ func TestDefaultEventProcessor_ProcessImpression(t *testing.T) {
 
 	assert.Equal(t, 1, processor.EventsCount())
 
-	time.Sleep(200 * time.Millisecond)
+	time.Sleep(2000 * time.Millisecond)
```

Comment:
Is this because the test is flaky?

Reply:
yes, i had instances where it failed, increasing the timeout made it more stable.